lib/pbio/platform/ev3/platform.c

```diff
@@ -422,6 +422,12 @@ static const char *const panic_types[] = {
     "Prefetch Abort",
     "Data Abort",
 };
+enum {
+    EV3_PANIC_UNKNOWN,
```

Comment:
Would be nice to share these defs with `lib/pbio/platform/ev3/exceptionhandler.S` instead of redefining them. Or at least add a comment explaining where the values come from.

(also could use some blank lines before and after to keep things spaced out.)